components/CollectiveNavbarActionsMenu.js

```diff
@@ -134,12 +160,68 @@ const CollectiveNavbarActionsMenu = ({
               )}
               {hasRequestGrant && (
                 <MenuItem py={1}>
-                  <Dollar size="20px" color="#304CDC" />
-                  <P my={2} fontSize="13px" color="black.800">
+                  <StyledLink
+                    as={Link}
+                    route="create-expense"
+                    params={{ collectiveSlug: collective.slug }}
+                    p={ITEM_PADDING}
+                  >
+                    <Dollar size="20px" color="#304CDC" />
                     <FormattedMessage id="ExpenseForm.Type.Request" defaultMessage="Request Grant" />
-                  </P>
+                  </StyledLink>
+                </MenuItem>
+              )}
+              {callsToAction.hasManageSubscriptions && (
+                <MenuItem>
+                  <StyledLink
+                    as={Link}
+                    route="recurring-contributions"
+                    params={{ slug: collective.slug }}
+                    p={ITEM_PADDING}
+                  >
+                    <Receipt size="20px" color="#304CDC" />
+                    <FormattedMessage id="menu.subscriptions" defaultMessage="Manage Contributions" />
+                  </StyledLink>
                 </MenuItem>
               )}
+              {callsToAction.hasContribute && (
+                <MenuItem py={1}>
+                  <StyledLink as={Link} route={contributeRoute} params={contributeRouteParams} p={ITEM_PADDING}>
+                    <Planet size="20px" color="#304CDC" />
+                    <FormattedMessage id="menu.contributeMoney" defaultMessage="Contribute Money" />
+                  </StyledLink>
+                </MenuItem>
+              )}
+              {callsToAction.addFunds && (
+                <Fragment>
+                  <MenuItem py={1}>
+                    <StyledButton p={ITEM_PADDING} onClick={() => showAddFundsModal(true)} isBorderless>
+                      <AttachMoney size="20px" color="#304CDC" />
+                      <Span>
+                        <FormattedMessage id="menu.addFunds" defaultMessage="Add funds" />
+                      </Span>
+                    </StyledButton>
+                  </MenuItem>
+                  <AddFundsModal collective={collective} show={hasAddFundsModal} setShow={showAddFundsModal} />
```

Comment:
`AddFundsModal` seems to have `host` prop required, does it work without it?

Reply:
I had missed that, good catch - thank you!